lib/index.js

```diff
@@ -46,6 +46,20 @@ Customerio.prototype.loaded = function() {
 };
 
 /**
+ * Page.
+ *
+ * https://learn.customer.io/api/#apipageview_event
+ *
+ * @api public
+ * @param {Page} page
+ */
+
+Customerio.prototype.page = function(page) {
+  var name = page.name() || page.url();
```

Comment:
@maxence-charriere actually i think it's probably better to use `page.path()`? what is the recommend implementation if you were to use this `_cio.page()` method natively without segment?